client-tags/read.md

```diff
@@ -0,0 +1,73 @@
+---
+title: IRCv3 read client tag
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+---
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `+read` tag name. Instead, implementations SHOULD use the
+`+draft/read` tag name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use the unprefixed tag name.
+
+## Description
+The specification includes two client tags for message tracking: `delivered` and `read`. These tags provide for delivery and read receipts, respectively, and enable users to know when and by whom a message has been delivered to or read.
```

Comment:
But #346 does it?
I think they should be combined in the same doc, if they are so connected.